examples/unconditional_image_generation/train_unconditional.py

```diff
@@ -92,7 +92,7 @@ def parse_args():
     parser.add_argument(
         "--resolution",
         type=int,
-        default=64,
+        default=-1,
```

Comment:
Not a fan of changing the default here - examples are meant to be easily readable. A default resolution of -1 is impossible to understand for people